### ISSUE TITLE
role='flatdoc': flatdoc is not valid value of attribute 'role'

### ISSUE THREAD

**Issue by PaparajuGowd** [rstacruz/flatdoc#104]
my website is failing the accessibility test. due to role flatdoc. do we have any other option to use flatdoc with role setting
